src/screenComponents/shipsLogInternControl.cpp

```diff
@@ -0,0 +1,85 @@
+#include "playerInfo.h"
+#include "spaceObjects/playerSpaceship.h"
+#include "shipsLogInternControl.h"
+
+#include "gui/gui2_panel.h"
+#include "gui/gui2_advancedscrolltext.h"
+
+ShipsLogIntern::ShipsLogIntern(GuiContainer* owner)
```

Comment:
It is a copy &paste indeed. I don't know now how to do a single class but I will learn !